clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -257,6 +257,8 @@ public class VenicePushJob implements AutoCloseable {
    */
   public static final String VENICE_DISCOVER_URL_PROP = "venice.discover.urls";
 
+  public static final String VENICE_CHILD_CONTROLLER_PROP = "venice.child.controller.urls";
```

Comment:
In addition to using child controller for request topic calls, we'll also need to make this system store as Active/Active since we want the heartbeats to be replicated to all colos. This would need changes in the store's initialization

Reply:
Right now, the store is already in the prod but not A/A, so we need to enable it by updating that store. So the initialization is more for test, so that we do not need to update that store anymore.

Reply:
Besides, for adopting D2. Does this mean we will rely on `D2ControllerClient` instead? Besides is this D2 migration for VPJ is a planned work or not?

Reply:
The initialization would be for tests - and any new deployments of Venice.

Yes, we need to use `D2ControllerClient`. D2 migration for VPJ is not planned work. I had done some work on it pre-OSS - but we couldn't merge it back then. It is currently a part of Bug Basher tasks but has not been picked up yet by anyone. (Full migration of VPJ to D2 is a bigger task and should not be a part of this PR)

The reason we need this to be AA is that the lingering job check is done by the parent controller and it can query routers from any child colo. Hence, the HB data needs to be replicated cross-colo and we can achieve it only via A/A.

Reply:
I have now merged the PR (#195) for using `D2ControllerClient` in VPJ. It seems we want to support D2 and URL based controller clients. Can you please try adding support for both - URL based and D2 based?